docs/docs/fault-domain-awareness.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Fault Domain Awareness
+---
+
+# Fault Domain Awareness
+
+## Overview
+
+A fault domain is a set of nodes that share similar failure and latency characteristics. Two nodes in the same fault domain are both affected by failure events within the domain. Placing nodes in more than one fault domain reduces the risk that a failure will affect both nodes.
```

Comment:
nodes usually refer to the server or VM running an instance. Would it be more clear to say:

> Placing _instances_ in more than one fault domain...

Reply:
I'm not sure where we got the definition of fault domain but it seemed narrow.   There are different types of fault domains... at a node level (if a node is lost all services on it are lost), at a rack level (if a rack is lost we lose all nodes on the rack), at a cluster level (if a router goes out all services are lost), at a region level.

from wikipedia:  a failure domain encompasses a section of a network that is negatively affected when a critical device or network service experiences problems. The size of a failure domain and its potential impact depends on the device or service that is malfunctioning

Reply:
@sascala it doesn't seem like any of my comments were addressed